internal/server/data/group.go

```diff
@@ -138,11 +154,13 @@ func RemoveUsersFromGroup(tx WriteTxn, groupID uid.ID, idsToRemove []uid.ID) err
 	return handleError(err)
 }
 
-// TODO: do this with a join in ListGroups and GetGroup
 func CountUsersInGroup(tx GormTxn, groupID uid.ID) (int64, error) {
 	db := tx.GormDB()
 	var count int64
-	err := db.Table("identities_groups").Where("group_id = ?", groupID).Count(&count).Error
+	err := db.
+		Table("identities_groups").
+		Where("group_id = ?", groupID).
```

Comment:
I think we need to count distinct users now.. so `count(distinct user_id)` should work.